src/command/use_.rs

```diff
@@ -4,40 +4,43 @@
 
 use semver::VersionReq;
 
+use notion_core::env;
 use notion_core::serial::version::parse_requirements;
 use notion_core::session::{ActivityKind, Session};
-use notion_fail::Fallible;
+use notion_fail::{Fallible, ResultExt};
 
 use Notion;
 use command::{Command, CommandName, Help};
 
+use std::fs::File;
+use std::io::Write;
 use std::process::exit;
 
 #[derive(Debug, Deserialize)]
 pub(crate) struct Args {
     arg_version: String,
-    flag_global: bool,
+    flag_save: bool,
 }
 
 pub(crate) enum Use {
     Help,
     Global(VersionReq),
-    Local(VersionReq),
+    Save(VersionReq),
 }
 
 impl Command for Use {
     type Args = Args;
 
     const USAGE: &'static str = "
-Activate a particular toolchain version
+Select a particular toolchain version
 
 Usage:
     notion use [options] <version>
     notion use -h | --help
 
 Options:
     -h, --help     Display this message
-    -g, --global   Activate the toolchain globally
+    -s, --save     Select the toolchain for the current Node project
```

Comment:
This may be good enough to start dogfooding, but I think we need more thought/design around command syntax, maybe some scratch doc with thoughts from this morning's meeting.

I like that we are preserving some nice familiar aspects of `nvm` syntax, but it doesn't have the concept of local project config, so that syntax is getting stretched to account for the Notion world. That said, I don't have a better idea for how to handle this at the moment.

Reply:
Agreed, this part of the design hasn't gelled yet.